### PR TITLE
GLT-2753: renamed library dilutions to library aliquots

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: plain-3-0-libraries.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Table of Contents
    <li><a href="#libraries-receipt">Receiving Libraries</a></li>
    <li><a href="#libraries-qc">Adding Library QCs</a></li>
    <li><a href="#boxes">Scanning libraries into your outbox</a></li>
-   <li><a href="#libraries-to-dilutions">Propagating libraries to dilutions</a></li>
-   <li><a href="#libraries-dils-to-pools">Creating Pools</a></li>
+   <li><a href="#libraries-to-library-aliquotss">Propagating libraries to library aliquots</a></li>
```
Comment (avarsava) [inline plain-3-0-libraries.md]: "aliquotss"?

File: 0-original-walkthrough.md [hunk 2]
```diff
@@ -758,48 +758,46 @@ e.g. `PROJ_P1_Li`.
...
 
-Dilutions are a bit ephemeral in MISO: there is no list for all the dilutions.
-To see the dilutions for a library, view the library page. At present,
-Dilutions cannot be saved in Boxes.
-
 ## 7.2 Pools
 Pools are the last step for libraries before sequencing and represent the
 entity that is loaded onto the flowcell lane or SMRTcell. A "pool" can have one or more
-library dilutions in it. They are equivalent to "worksets" in Geospiza, but are
+library aliquots in it. They are equivalent to "worksets" in Geospiza, but are
```
Comment (callunity) [inline 0-original-walkthrough.md]: This line about pools being `equivalent to "worksets" in Geospiza` may have outlived its usefulness... especially as MISO now has worksets.

Comment (djcooke) [reply 0-original-walkthrough.md]: Didn't see that. Will happily remove, along with the other couple of references to Geospiza!

File: plain-3-0-libraries.md (hunk 1 above)
Comment (djcooke) [reply plain-3-0-libraries.md]: fixed

**Pull request merged**